editor/components/block-list/block.js

```diff
@@ -231,7 +231,10 @@ export class BlockListBlock extends Component {
 			return;
 		}
 
-		target.focus();
+		// If there is a contenteditable element, it will move focus by itself.
+		if ( ! target.querySelector( '[contenteditable="true"]' ) ) {
+			target.focus();
+		}
```

Comment:
This is the only change that's bothering me in this PR because it introduces a special case in how blocks focus is handled.

I'm thinking all this behavior `focusTabbable when a new block is selected` could be moved into a separate component and also handle the usecase we need for this PR: Focus the first tabbable element at the right position :)

---
Anyway, might not be that easy to do, so maybe for now, just consolidate the test of the presence of a TinyMCE editor to match the test already done here https://github.com/WordPress/gutenberg/pull/4775/files#diff-bc66d5701a2fea1111bb8f1bf0150910R244 `const editor = tinymce.get( target.getAttribute( 'id' ) );`

Reply:
Also, I believe this assumes that the TinyMCE will focus itself which I believe is not always what we want. Imagine inserting a block containing a plainText followed by the richText. TinyMCE won't be focused in that case.

I believe this is only necessary if we use the `onSplit` behavior of the RichText element.